nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/HandleHttpRequest.java

```diff
@@ -220,6 +221,13 @@
             .allowableValues(CLIENT_NONE, CLIENT_WANT, CLIENT_NEED)
             .defaultValue(CLIENT_NONE.getValue())
             .build();
+    public static final PropertyDescriptor TIMEOUT = new PropertyDescriptor.Builder()
+            .name("Timeout")
+            .description("The timeout to set when handling requests")
+            .required(false)
```

Comment:
I typically recommend setting required = true if there is a default value. This is just because a value will always be present and as a result this flag doesn't really mean a whole lot. So I use required = true for consistency.